stetho/src/main/java/com/facebook/stetho/inspector/elements/android/HighlightableDescriptor.java

```diff
@@ -13,7 +13,7 @@
 
 import javax.annotation.Nullable;
 
-interface HighlightableDescriptor {
+public interface HighlightableDescriptor<E> {
```

Comment:
do we need to make this public? seems like this is something for a later diff, right?